va/va.h

```diff
@@ -4187,6 +4187,25 @@ Synchronization
  * This function blocks until all pending operations on the render target
  * have been completed.  Upon return it is safe to use the render target for a
  * different picture.
+ *
+ * Possible errors:
+ * - \ref VA_STATUS_ERROR_INVALID_CONTEXT: This error occurs when the function is provided with an invalid context.
+ *   It is typically due to the context not being properly initialized or already being freed.
+ *   To address this, ensure that the context is correctly initialized and has not been freed.
+ * - \ref VA_STATUS_ERROR_INVALID_SURFACE: This error is triggered when the supplied render target is invalid,
+ *   possibly due to being freed or corrupted. To resolve this, ensure the usage of the correct render target
+ *   and verify its validity and integrity.
+ * - \ref VA_STATUS_ERROR_OPERATION_FAILED: When the attempt to retrieve the status report fails or related work
+ *   on the current surface fails, this error is generated. In such cases, the client can proceed with subsequent
+ *   frames by ignore the error, but it may involve incorrect result like corruption.
```

Comment:
it means "operation failed" could be ignored by application

Reply:
VA_STATUS_ERROR_OPERATION_FAILED is caused by status report fail which cannot be used to determine whether the result is correct. So declare here that APP can ignore this error but may meet incorrect result.

And will add more detail like _"To init new context to avoid protentional incorrect result."_